docs/content/preview/explore/transactions/distributed-transactions-ysql.md

```diff
@@ -201,7 +209,7 @@ where transaction_mode is one of:
   [ NOT ] DEFERRABLE
 ```
```

Comment:
In the "Run a transaction" section:

_This is shown diagrammatically in the following illustration._ --> The following diagram shows the sequence of events that occur when a transaction is running:
(or something like that)